locale/nl.po

```diff
@@ -4223,7 +4223,7 @@ msgid ""
 "When you have finished configuring the keys, go right to the next Action "
 "Point"
 msgstr ""
-"Indien het de de toetsconfiguratie is voltooid, ga rechts naar volgende"
+"Indien de toetsconfiguratie is voltooid, ga rechts naar het volgende actiepunt"
```

Comment:
This was the only line that seems to be longer than the other ones, not sure if that causes any problems? I guess launchpad formats them this way? It's right at the start of the tutorial.